docs/fwupdplugin.toml.in

```diff
@@ -46,6 +46,7 @@ content_files = [
   "tutorial.md",
   "hsi.md",
   "ds20.md",
+  "hwids.md",
```

Comment:
think you want to also update `index.html`

Reply:
I guess; fixed.